src/openrct2/ride/Vehicle.cpp

```diff
@@ -5410,7 +5410,9 @@ void Vehicle::UpdateSound()
     if (rideEntry == nullptr)
         return;
 
-    const auto& carEntry = rideEntry->Cars[vehicle_type];
+    // Always use the head car's sound data (Some of the other vehicle subtypes have improperly set data)
```

Comment:
We should probably add a note to the CarEntry struct that `friction_sound_id` and `CAR_ENTRY_FLAG_RIDERS_SCREAM` are only valid for the FrontCar/DefaultCar